modules/structs/repo.go

```diff
@@ -239,10 +239,16 @@ type CreateBranchRepoOption struct {
 	// unique: true
 	BranchName string `json:"new_branch_name" binding:"Required;GitRefName;MaxSize(100)"`
 
+	// Deprecated: true
 	// Name of the old branch to create from
 	//
 	// unique: true
 	OldBranchName string `json:"old_branch_name" binding:"GitRefName;MaxSize(100)"`
+
+	// Name of the old branch/tag/commit to create from
+	//
+	// unique: true
+	OldRefName string `json:"old_ref_name" binding:"GitRefName;MaxSize(100)"`
```

Comment:
Now we have two fields binding to `GitRefName`, what will happen?

Reply:
Emmm...,But `BranchName` also binding to `GitRefName`

Reply:
Now we have three fields binding to `GitRefName`, it should be

Reply:
This is a validation binding not a name binding